configure.ac

```diff
@@ -8,7 +8,7 @@ dnl ============================================================================
 dnl Custom macro definitions.
 
 dnl JE_CONCAT_VVV(r, a, b)
-dnl 
```

Comment:
@jasone: I think you mentioned the space after dnl was required, but couldn't find exactly why. Is this going to be an issue?